Packet++/header/SSLCommon.h

```diff
@@ -388,7 +388,7 @@ namespace pcpp
 	/**
 	 * SSL/TLS symmetric encryption algorithms
 	 */
-	enum SSLSymetricEncryptionAlgorithm
+	enum SSLSymmetricEncryptionAlgorithm
```

Comment:
enums are more difficult to clone and dperecate. I suggest we just leave it with the typo. What do you think?

Reply:
We can duplicate values but probably not worth it 🤔. I think adding exception is easier.